From 1e4c5cded117e95d380629bea44009bb3194113f Mon Sep 17 00:00:00 2001
From: Alexander Filonenko <alexf@motorola.com>
Date: Fri, 8 Apr 2016 15:52:38 -0500
Subject: [PATCH 849/959] IKKRNBSP-3823 input: synaptics_dsx: expose F54.d17
 presence flag

Exspose F54.q16.has_data17 flag which determines presence of
F54.d17 register.

Change-Id: If4a9aa109a5b5c5bcd39fc799ecd0162dce54611
Signed-off-by: Alexander Filonenko <alexf@motorola.com>
Reviewed-on: http://gerrit.mot.com/842823
SLTApproved: Slta Waiver <sltawvr@motorola.com>
SME-Granted: SME Approvals Granted
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 drivers/input/touchscreen/synaptics_dsx_test_reporting.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/input/touchscreen/synaptics_dsx_test_reporting.c b/drivers/input/touchscreen/synaptics_dsx_test_reporting.c
index 8a877d06621..65dd9ec2882 100644
--- a/drivers/input/touchscreen/synaptics_dsx_test_reporting.c
+++ b/drivers/input/touchscreen/synaptics_dsx_test_reporting.c
@@ -1970,6 +1970,7 @@ show_prototype(has_noise_mitigation2)
 show_prototype(has_noise_state)
 show_prototype(has_energy_ratio_relaxation)
 show_prototype(number_of_sensing_frequencies)
+show_prototype(has_data17)
 show_prototype(q17_num_of_sense_freqs)
 show_store_prototype(no_relax)
 show_store_prototype(no_scan)
@@ -2148,6 +2149,7 @@ static struct attribute *attrs[] = {
 	attrify(has_noise_state),
 	attrify(has_energy_ratio_relaxation),
 	attrify(number_of_sensing_frequencies),
+	attrify(has_data17),
 	attrify(q17_num_of_sense_freqs),
 	attrify(f55_q2_has_single_layer_multitouch),
 	NULL,
@@ -3449,6 +3451,7 @@ simple_show_func_unsigned(query, has_noise_mitigation2)
 simple_show_func_unsigned(query, has_noise_state)
 simple_show_func_unsigned(query, has_energy_ratio_relaxation)
 simple_show_func_unsigned(query12, number_of_sensing_frequencies)
+simple_show_func_unsigned(query16, has_data17)
 simple_show_func_unsigned(query17, q17_num_of_sense_freqs)
 show_store_func_unsigned(data, reg_4, d4_inhibit_freq_shift)
 show_store_func_unsigned(data, reg_4, d4_baseline_sel)
-- 
2.11.0

